chargeio/chargeio.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <Version>1.1.0</Version>
+    <Authors>Victor Ude</Authors>
+    <Company>AffiniPay Holdings LLC</Company>
+    <Product>affinipay.com</Product>
+    <PackageId>chargeio-net</PackageId>
+    <AssemblyVersion>1.1.0.0</AssemblyVersion>
+    <FileVersion>1.1.0.0</FileVersion>
+    <NeutralLanguage>en</NeutralLanguage>
+    <Description>.NET client library for the chargeIO payment gateway</Description>
```

Comment:
Should refer to "AffiniPay Payment Gateway" now, rather than "chargeIO payment gateway"